helm/nats-box/templates/nats-box.yaml

```diff
@@ -2,22 +2,40 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: nats-box
-  namespace: { { $.Release.Namespace | default "default" } }
+  namespace: {{ $.Release.Namespace | default "default" }}
   labels:
     app: nats-box
   annotations:
     sidecar.istio.io/inject: 'false'
 spec:
+  {{- if false }}
+  volumes:
+    - name: creds-volume
+      secret:
+        secretName: stan-creds
+  {{- end }}
```

Comment:
Removeable

Reply:
i'd like to keep it, as it is required for surveyor

Reply:
stan-creds is a a mount used by other processes

Reply:
Should rename to `nats-creds` then